### DISCUSSION TITLE
Start a dialogue when colliding with an entity

### DISCUSSION THREAD

**Discussion by fleggs** [AlmasB/FXGL#1162] [Q&A]
Hi,
I am trying to create a game like Zephyria, but I'm struggling with the dialogues. I've created a JSON with the dialogue editor and I want to start it, if the player collides with the entity. Wasn't able to figure out where I should mention the JSON and which method to use. 
All help is appreciated.

**Comment by AlmasB** [comment]
Hi, have a look at [this sample](https://github.com/AlmasB/FXGL/blob/dev/fxgl-samples/src/main/java/advanced/DialogueSample.java) for some pointers. Let me know if this helps.

**Reply by fleggs** [reply]
Thanks a lot, I've got it running. I think I forgot the Cutscene Service.
